packetbeat/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Beats dockerfile used for testing
+FROM ruflin/beat-tester
```

Comment:
It is here, but I actually didn't want to include it in this PR as it depends on my personal repo: https://hub.docker.com/r/ruflin/beat-testing/ Let me clean that up.